features/product/managing_product_variants/preventing_the_generation_of_products_variants_from_options_without_any_values.feature

```diff
@@ -0,0 +1,17 @@
+@managing_product_variants
+Feature:Preventing the generation of product variants from options without any values
+    In order to not be able to generate variants without any options values available
```

Comment:
Still too similar to the last line :) I would go with `In order to prevented from invalid variant generation`

Reply:
The other way round, the last sentence (I want to) should be changed.